examples/nlp/language_modeling/conf/megatron_gpt_inference.yaml

```diff
@@ -26,12 +26,13 @@ checkpoint_dir: null # checkpoint file dir. This is used to load the PTL checkpo
 checkpoint_name: null # PTL checkpoint file name, only used for PTL checkpoint loading
 hparams_file: null # model configuration file, only used for PTL checkpoint loading
 prompts: # prompts for GPT inference
-  - "Q: How are you?"
-  - "Q: How big is the universe?"
+  - "<extra_id_x>Q: How is <extra_id_0> created? A:"
+  - "<extra_id_x>Q: How big <extra_id_0> at night? A:"
```

Comment:
It seems painful to add `<extra_id_x>`? I think most people will want to just add their prompt in language?

Reply:
Yup! I think we need to revert this before actually merging.